dockworker.py

```diff
@@ -115,9 +115,27 @@ def create_notebook_server(self, base_path, container_name, container_config):
             rendered_command
         ]
 
+        volume_bindings = {}
+        volumes = []
+        if container_config.host_directories:
+            directories = container_config.host_directories.split(",")
+            for index, item in enumerate(directories):
+                directory = item.split(":")[0]
+                try:
+                    permissions = item.split(":")[1]
+                except IndexError:
+                    permissions = 'rw'
```

Comment:
We probably should make our default `'ro'` if they don't specify. If there's a compelling reason why not, like Docker's default being `'rw'` (I'm assuming so), then sure.